packages/@react-aria/utils/src/useViewportSize.ts

```diff
@@ -57,7 +57,7 @@ export function useViewportSize(): ViewportSize {
 
 function getViewportSize(): ViewportSize {
   return {
-    width: visualViewport?.width || window.innerWidth,
-    height: visualViewport?.height || window.innerHeight
+    width: (visualViewport && visualViewport.width) || window.innerWidth,
+    height: (visualViewport && visualViewport?.height) || window.innerHeight
```

Comment:
Why did you add the `visualViewport &&` on both, but only remove the `?` on one of these?